go.mod

```diff
@@ -4,6 +4,12 @@ go 1.12
 
 require (
 	github.com/aws/aws-sdk-go v1.26.6
-	github.com/mattermost/mattermost-server/v5 v5.18.0
-	github.com/pkg/errors v0.8.1
+	github.com/blang/semver v3.5.1+incompatible
```

Comment:
Nit pick: `github.com/blang/semver/v4` was just released which is go modules compatible.

Reply:
Thanks, now updated.